components/tabs/stories/tabs-horizontal.stories.js

```diff
@@ -30,11 +28,110 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
+const chromaticKitchenSink = ({
```

Comment:
```suggestion
const HorizontalTabsGroup = ({
```

I've noticed a move toward `chromaticKitchenSink` but this name is awfully generic. If we want to export the set to use elsewhere, we'd have to rename it on entry. I also love when variable names are as specific as possible to make it clear without documentation.